### PR TITLE
Add GIT_URL and GIT_SHA to docker labels

### PR DESCRIPTION
**Pull request by oshoval**
In order to be able to resolve for each image what is the git
commit it was built from using docker inspect.

Signed-off-by: Or Shoval <oshoval@redhat.com>

can be inspected by using

docker inspect --format "{{ index .Config.Labels \"multi.GIT_URL\"}}" <IMAGE>
docker inspect --format "{{ index .Config.Labels \"multi.GIT_SHA\"}}" <IMAGE>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,6 +7,8 @@ IMAGE_TAG          ?= latest
 OPERATOR_IMAGE     ?= kubevirt/hyperconverged-cluster-operator
 REGISTRY_NAMESPACE ?=
 
+SHA:=$(shell git describe --no-match  --always --abbrev=40 --dirty)
```
Comment (djzager) [inline Makefile]: nit: you should put this up with, and aligned, with the rest.

File: build/Dockerfile [hunk 2]
```diff
@@ -19,3 +19,9 @@ COPY --from=builder /go/src/github.com/kubevirt/hyperconverged-cluster-operator/
 COPY --from=builder /go/src/github.com/kubevirt/hyperconverged-cluster-operator/templates/olm-catalog/kubevirt-hyperconverged/VERSION/kubevirt-hyperconverged-operator.VERSION.clusterserviceversion_merger.yaml.in $CSV_TEMPLATE
 ENTRYPOINT ["/usr/local/bin/entrypoint"]
 USER ${USER_UID}
+
+ARG git_url=https://github.com/kubevirt/hyperconverged-cluster-operator
```
Comment (oshoval) [inline build/Dockerfile]: will add here .git since sometimes git clone without .git have problems

File: Makefile (hunk 1 above)
Comment (oshoval) [reply Makefile]: thanks will do

**Comment by rthallisey** [comment]
Seems like SSP pod is failing to start in prow
```
   Unexpected error:
      
      <*errors.errorString | 0xc0006241a0>: {
          s: "No ready pod listening on the service.",
      }
      No ready pod listening on the service.
  occurred 
```

**Comment by oshoval** [comment]
in case the cert issue of ci/prow/hco-e2e-aws is known (which is the case i think, but please approve)
can we override it ?

**Comment by rthallisey** [comment]
ci test please

**Pull request merged**